### PR TITLE
fixing cred_preview issue in issuing credential offer

### PR DESCRIPTION
**Pull request by spsingh559**
In Issuing credential offer for Acme, cred_preview variable is missing and due to this acme agent will throw " Acme | Error during POST /issue-credential/send-offer: 422, message='Unprocessable Entity', url=URL('http://192.168.65.3:8041/issue-credential/send-offer')"

cred_preview_error

So to fix this issue, I have added the cred_preiew object and add that property in offer_request object.

### CODE REVIEW

**Comment by sklump** [comment]
I can take this tomorrow morning

**Comment by swcurran** [comment]
@spsingh559 --- note the problem with the DCO - the signing of the commit (using "-s" on the commit command, if you are using command line git).  Can you look at the "details" like beside that and see if you can fix it?  You may have to do another PR if this cannot be fixed in this PR.  Sorry about that -- a bump in the road on the first commit to a Hyperledger Repo.

If you have trouble, we can connect on RocketChat so I know how you are using git and then what you need to do.

@ianco - can you please review the change?

Thanks!

**Comment by spsingh559** [comment]
Thanks Stephen, I have followed the step mentioned here in DCO Details https://github.com/hyperledger/aries-cloudagent-python/pull/549/checks?check_run_id=752742907

Shall I create a new pull request and repeat the process again.

**Comment by sklump** [comment]
109 files changed? Surely it must be simpler to start from master on a new PR?

File: demo/AcmeDemoWorkshop.md [hunk 1]
```diff
@@ -232,10 +232,17 @@ with the following code:
                     "date": date.isoformat(date.today()),
                     "position": "CEO"
                 }
+                cred_preview = {
```
Comment (sklump) [inline demo/AcmeDemoWorkshop.md:235]: agent.cred_attrs.items() here I think?

File: demo/AcmeDemoWorkshop.md [hunk 2]
```diff
@@ -232,10 +232,17 @@ with the following code:
...
+                    "@type": CRED_PREVIEW_TYPE,
+                    "attributes": [
+                        {"name": n, "value": v} for (n, v) in cred_attrs.items()
```
Comment (sklump) [inline demo/AcmeDemoWorkshop.md:238]: agent.cred_attrs.items() here I think?

Comment (sklump) [inline demo/AcmeDemoWorkshop.md:238]: `cred_attrs.items()` doesn't work here - prefer `agent.cred_attrs[credential_definition_id].items()`

**Comment by sklump** [comment]
Signed and repackaged as https://github.com/hyperledger/aries-cloudagent-python/pull/550: thanks, good catch

**Pull request closed without merging**